src/sonic_ax_impl/mibs/vendor/cisco/ciscoEntityFRUControlMIB.py

```diff
@@ -0,0 +1,106 @@
+import subprocess
+import re
+
+from sonic_ax_impl import mibs
+from ax_interface import MIBMeta, ValueType, MIBUpdater, MIBEntry, SubtreeMIBEntry
+from ax_interface.encodings import ObjectIdentifier
+
+import pdb
+
+class PowerStatusHandler:
+    """
+    Class to handle the SNMP request
+    """
+    def __init__(self):
+        """
+        init the handler
+        """
+
+        # Holds the number of supported PSU
+        self.num_psus = self._getNumOfSupportedPsu()
+
+    def _getNumOfSupportedPsu(self):
+        """
+        :return: the number of supported PSU
+        """
+        try:
+            output = subprocess.check_output(['psuutil', 'numpsus'])
```

Comment:
>check_output [](start = 32, length = 12)

Use a library instead of subprocess? If it is in plan, please add a TODO comment.

Reply:
Do you mean to use the python plugin instead of subprocess?
If yes, such discussion is considering in a following link:
https://github.com/Azure/sonic-buildimage/pull/1170/files/1da68a14070bf2718739841898ac5d18fe3430e8#diff-d354c022e8bdbc0e1323fcca2eaa0ee2